chart/templates/etcd/storage/localpv-storageclass.yaml

```diff
@@ -0,0 +1,16 @@
+{{ if and (index .Values "localpv-provisioner" "enabled") (and .Values.etcd.persistence.enabled (ne (tpl (.Values.etcd.localpvScConfig.name) .) "")) }}
```

Comment:
Can we "create" an error error if .Values.etcd.localpvScConfig.name == "" ?

Reply:
It will anyhow fail ultimately like this.

`Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: could not get information about the resource StorageClass "" in namespace "": resource name may not be empty`

Reply:
But that's during installation rather than during templating?
You can use this: https://github.com/openebs/mayastor-extensions/blob/9bc0ebfb2725e6a60d78f369e5011ea41165801c/chart/templates/_helpers.tpl#L125

Reply:
Done.